src/control/server/server.go

```diff
@@ -178,6 +179,13 @@ func Start(log *logging.LeveledLogger, cfg *Configuration) error {
 			srvCfg.WithEnvVars(envVar)
 		}
 
+		if i == 0 {
+			netDevClass, err = netdetect.GetDeviceClass(srvCfg.Fabric.Interface)
+			if err != nil {
+				return err
+			}
+		}
```

Comment:
I think that we need to sanity-check that all of the ioservers have the same class of interface and fail with an error if not. This should probably be a configuration fault, similar to the ones in https://github.com/daos-stack/daos/blob/master/src/control/server/faults.go#L61